modules/core/src/main/scala/org/scalasteward/core/forge/ForgeType.scala

```diff
@@ -18,12 +18,16 @@ package org.scalasteward.core.forge
 
 import cats.Eq
 import cats.syntax.all._
+import org.http4s.Uri
 import org.http4s.syntax.literals._
+import org.scalasteward.core.application.Config.ForgeCfg
 import org.scalasteward.core.forge.ForgeType._
 import org.scalasteward.core.util.unexpectedString
 
 sealed trait ForgeType extends Product with Serializable {
   def publicWebHost: Option[String]
+  val diffs: DiffUriPattern
```

Comment:
**quibble:** Can we add some ScalaDocs here to explain what this method should do?

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org) standard</sub>

Reply:
Thanks, done with 905250cc0280c680a2c8016eb2c8cc22f87398a3 !